package.json

```diff
@@ -37,7 +37,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@razee/kubernetes-util": "0.0.10",
-    "@razee/razeedeploy-core": "^0.7.0",
+    "@razee/razeedeploy-core": "^0.7.1",
```

Comment:
based on my working knowledge of semver, i think we want to go to 0.8.0

```
Semantic Versioning Rules:
MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backwards-compatible manner
PATCH version when you make backwards-compatible bug fixes.
```

Reply:
I'm fine with that. 0.8.0 it is.